src/lv_message_efficient.h

```diff
@@ -86,13 +84,14 @@ namespace grpc_labview
             // copy into LVCluster
             if (numElements != 0)
             {
-                NumericArrayResize(0x08, 1, reinterpret_cast<void*>(const_cast<char*>(_lv_ptr)), numElements);
```

Comment:
NOTE: This was over allocating for any element type that wasn't 8 bytes (so basically anything other than doubles).